### DISCUSSION TITLE
Creating an academic organization

### DISCUSSION THREAD

**Discussion by IINemo** [community/community#37621] [Other Feature Feedback, Questions, & Ideas] [answered]
I would like to create an academic organization for our department. We would like to have the shared space for our research projects. I know how to create an individual academic account: you need to register and provide the proof of your academic status with ID. However, I do not understand how to create an academic account for the whole organization. Could you help?

**Comment by jfullstackdev** [comment]
you might be referring to GitHub Developer Pack for Students ? 

because even without that, you can create a GitHub Org as the central space for your research projects

**Comment by abhinit21** [comment, marked as answer]
@IINemo 

You can use GitHub Organizations for free. Your team can collaborate on GitHub by using an organization account, which serves as a container for your shared work and gives the work a unique name and brand. You can also manage, collaborate between teams inside origination.

Check out how to create GitHub Organization [here](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/creating-a-new-organization-from-scratch)

**Comment by IINemo** [comment]
@jdevstatic @abhinit21 Thank you for your replies, that was very helpful! Do you know whether GitHub provides free Team accounts to academia?

![image](https://user-images.githubusercontent.com/21058413/199210256-f431c941-d25a-4a57-9d07-dcce25a08ca3.png)

**Reply by abhinit21** [reply]
According to this github documentation you can get discount based on different scenarios

👉  [link](https://docs.github.com/en/billing/managing-billing-for-your-github-account/discounted-subscriptions-for-github-accounts)
